src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createTextNode = (type, parentNode, classes, text) => {
+  const node = document.createElement(type);
+
+  node.classList.add(...classes);
+  node.textContent = text;
+  parentNode.append(node);
+
+  return node;
+};
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
```

Comment:
The `promise1` is immediately resolved and its value is a constant string. If there's no specific reason to use a Promise here, you could just use the string directly.